build.sbt

```diff
@@ -14,6 +14,32 @@ val http4sVersion = "0.21.21"
 val circeVersion = "0.13.0"
 val sttpVersion = "1.7.2"
 
+val GraalVM11 = "graalvm-ce-java11@20.3.0"
+
+ThisBuild / scalaVersion := Scala_212
+ThisBuild / crossScalaVersions := Seq(Scala_212, Scala_213)
+ThisBuild / githubWorkflowJavaVersions := Seq(GraalVM11)
+ThisBuild / githubWorkflowBuild := Seq(
+  WorkflowStep.Sbt(List("test", "mimaReportBinaryIssues"))
+)
+
+//sbt-ci-release settings
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+ThisBuild / githubWorkflowPublishTargetBranches := Seq(
+  RefPredicate.StartsWith(Ref.Branch("main")),
```

Comment:
we're using `master` for now, let's add another line for that (so we'll have both, it'll be a smoother transition when the rename happens)